website/docs/r/private_endpoint.html.markdown

```diff
@@ -199,7 +199,9 @@ An `ip_configuration` supports the following:
 
 * `private_ip_address` - (Required) Specifies the static IP address within the private endpoint's subnet to be used. Changing this forces a new resource to be created.
 
-* `subresource_name` - (Required) Specifies the subresource this IP address applies to. `subresource_names` corresponds to `group_id` and in this context is also used for `member_name`. Changing this forces a new resource to be created.
+* `subresource_name` - (Required) Specifies the subresource this IP address applies to. `subresource_names` corresponds to `group_id`. Changing this forces a new resource to be created.
+
+* `member_name` - (Optional) Specifies the member name of this IP address applies to. `member_name` corresponds to `group_id`, the default value is `default`.
```

Comment:
if `member_name` corresponds to `group_id` why is the property not called group_id?